x-pack/plugins/maps/public/util.ts

```diff
@@ -117,41 +101,3 @@ export function getGlyphUrl(): string {
 export function isRetina(): boolean {
   return window.devicePixelRatio === 2;
 }
-
-export async function fetchGeoJson(
```

Comment:
thx for moving. This logic was pushed into ems-client, so there was no re-use anymore between `kibana_regionmap_source` and `ems_layer_source`